Minesweeper.java

```diff
@@ -0,0 +1,99 @@
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionListener;
+import java.util.*;
+import java.awt.event.*;
+
+public class Minesweeper {
+
+    // The size in pixels for the frame.
+    private static final int SIZE = 500;
+    private JFrame  frame;
+
+    private final ActionListener actionListener = actionEvent -> {
+        Object source = actionEvent.getSource();
+    };
+
+    private class Cell extends JButton {
```

Comment:
what is the advantage of `Cell`s extending `JButton` over having them be a standard class with a `JButton button` attribute? Personally I think the latter is more readable but you can convince me otherwise

Reply:
thats the standard swing notation/syntax from what i've seen in the documentation/other projects, that way we can do whatever we want on a "Cell" the same way as if it is a button itself.